web3/utils/normalizers.py

```diff
@@ -137,6 +138,8 @@ def abi_ens_resolver(w3, abi_type, val):
             raise InvalidAddress("Could not look up name, because no web3 connection available")
         elif w3.ens is None:
             raise InvalidAddress("Could not look up name, because ENS is set to None")
+        elif int(w3.net.version) is not 1 and not isinstance(w3.ens, StaticENS):
+            raise InvalidAddress("Could not look up name, because web3 is not connected to mainnet")
```

Comment:
Would you add the name `val` to this message and the ones above it? Something like:

`"Could not look up name %r, because web3 is not connected to mainnet" % val`